components/hab/src/command/supportbundle.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2016-2018 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::fs::File;
+use std::env;
+use std::path::{Path, MAIN_SEPARATOR};
+use std::process;
+use tar;
+use flate2::Compression;
+use flate2::write::GzEncoder;
+use chrono::Local;
+use error::Result;
+use hcore::fs::FS_ROOT_PATH;
+use common::ui::{Status, UIWriter, UI};
+
+pub fn start(ui: &mut UI) -> Result<()> {
+    let dt = Local::now();
+    let tarball_name = format!("support-bundle-{}.tar.gz", dt.format("%Y%m%d%H%M%S"));
```

Comment:
This is probably fine for right now, but I can envision a potential future where we're able to execute this remotely using the new command-and-control functionality and maybe even gather all the tarballs back to the command machine. 

What do you think about incorporating something like the hostname into this filename? Even without the command-and-control aspect, I imagine people will be running this across their Habitat fleet anyway, which means they'll want some kind of machine-specific indicator in the filename anyway.